unbound/USAGE-server.md

```diff
@@ -0,0 +1,20 @@
+
+# Usage of the DNS server
+
+[Unbound DNS server](https://github.com/NLnetLabs/unbound) was configured with tls connection using the post quantum [openssl](https://github.com/open-quantum-safe/openssl) variant.
+## Installation
+Assuming you have docker [installed](https://docs.docker.com/install) on your machine all command below will launch dns server docker.
+
+Run Unbound DNS container:
+```bash
+    docker network create unbound-test
+    docker run --network unbound-test --interactive --publish=853:853 --tty --hostname unbound --name unbound -it openquantumsafe/unbound
+```
+After running all the command above a container will open with unbound running configure with dns-over-tls. 
+Before running unbound, a certificate is needed. Therefore a self sign certificate will be generate using the [unbound.sh](unbound-docker/unbound.sh) and ask for input.
+
+In the file [unbound.sh](unbound-docker/unbound.sh#L47) the environment variable  TLS_DEFAULT_GROUPS is set "p384_kyber768:X25519" to force for p384_kyber768 key exchange. Other key exchange algorithms can be used, find more algorithm in the [list of available post quantum key exchange algorithms](https://github.com/open-quantum-safe/boringssl#key-exchange). 
```

Comment:
Also here, it may be easier to provide an example how to set `TLS_DEFAULT_GROUPS` via command line instead, e.g., `docker run -e TLS_DEFAULT_GROUPS=kyber1024 --network unbound-test --interactive --publish=853:853 --tty --hostname unbound --name unbound -it openquantumsafe/unbound`. Please check whether this command indeed works OK as I only added it as an example.

Reply:
Yes I test the command and it work, I input it in both USAGE file for a user to know how to change the key exchange algorithm. Thank you for the help